domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/dto/CreateCompletedReviewDto.kt

```diff
@@ -10,6 +10,6 @@ data class CreateCompletedReviewDto(
   val conductedAt: LocalDate,
   val conductedBy: String?,
   val conductedByRole: String?,
-  val prisonerReleaseDate: LocalDate,
+  val prisonerReleaseDate: LocalDate?,
```

Comment:
Not all prisoners will have a release date, so have needed to make this field nullable